src/Azure.Functions.Cli/Azure.Functions.Cli.csproj

```diff
@@ -165,8 +165,8 @@
   <ItemGroup Condition="'$(NoWorkers)' != 'true'">
     <PackageReference Include="Microsoft.Azure.Functions.JavaWorker" Version="2.1.0-SNAPSHOT" />
     <PackageReference Include="Microsoft.Azure.Functions.NodeJsWorker" Version="3.0.0" />
-    <PackageReference Include="Microsoft.Azure.Functions.PowerShellWorker.PS7.0" Version="4.0.1273" />
-    <PackageReference Include="Microsoft.Azure.Functions.PowerShellWorker.PS7.2" Version="4.0.1274" />
+    <PackageReference Include="Microsoft.Azure.Functions.PowerShellWorker.PS7.0" Version="4.0.1570" />
```

Comment:
Please see https://github.com/Azure/azure-functions-core-tools/pull/2971 and https://github.com/Azure/azure-functions-core-tools/pull/2972. I have added a script to enforce the host & workers are always updated at the same time and should always match. This means language owners will never need to individually update their worker in this repo. Assuming y'all approve of my first PR, we can close this one.